src/pdlpack/pdlpack.py

```diff
@@ -261,60 +261,60 @@ def unescape(string):
         return re.sub(r'\\(?P<char>[/@:\\])', '\g<char>', string)
 
 def _raw_run_sql_query(sql, show_error):
-	"""
-	Runs a SQL query on the target platform DB
-	using the default command-line utility.
-	Very limited:
-	  - no text output with "new line" characters allowed
-	     @param sql query text to execute
-	     @param show_error displays the SQL error msg
-	"""
-	sqlcmd = 'psql'
-	delimiter = '|'
-
-	# Test the DB cmd line utility
-	std, err = subprocess.Popen(['which', sqlcmd], stdout=subprocess.PIPE,
-		                    stderr=subprocess.PIPE).communicate()
-
-	if std == '':
-	    _error("Command not found: %s" % sqlcmd, True)
-
-	# Run the query
-	global con_args
-	runcmd = [sqlcmd,
-		  '-h', con_args['host'].split(':')[0],
-		  '-d', con_args['database'],
-		  '-U', con_args['user'],
-		  '-F', delimiter,
+        """
```

Comment:
just git checkout out the original copy from master and make just the one line change. these huge diffs will confuse future collabs.....